ovon/dataset/pose_sampler.py

```diff
@@ -1,4 +1,4 @@
-from typing import List, Optional, Tuple
```

Comment:
@archana53 can you remove the incomplete files for now? For ex: the `instance_objectnav_generator.py` seems incomplete.

Reply:
Yeah. Got it